### PR TITLE
Add press page

### PR DESCRIPTION
**Pull request by dantesolis**
- [x] Add press page link to navigation menu
- [x] Add press page redirect

presspage2
presspage_1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,18 +1 @@
...
-
-# Ignore the build directory
-/build
-
-# Ignore cache
-/.sass-cache
-/.cache
-
-# Ignore .DS_store file
-.DS_Store
```
Comment (dantesolis) [inline .gitignore]: Hey, I fixed this mistake on another branch and pushed it already. Sorry about that.

File: source/assets/stylesheets/components/_cards.scss [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    }
+  }
+
+  .img-small {
+      width: 50px;
+      height: 50px;
+  }
+} /* end of card */
+
+@media(min-width:767px) {
```
Comment (dantesolis) [inline source/assets/stylesheets/components/_cards.scss]: Having a bit of a problem when scaling them on my browser?  @borisrorsvort  or anyone else mind helping?

Thanks 👍

File: .gitignore (hunk 1 above)
Comment (borisrorsvort) [inline .gitignore]: This should be fixed in this pr

File: Gemfile.lock [hunk 3]
```diff
@@ -219,6 +219,3 @@ DEPENDENCIES
   middleman-minify-html
   tzinfo-data
   wdm (~> 0.1.0)
-
```
Comment (borisrorsvort) [inline Gemfile.lock]: Please dont remove stuff like this

File: data/press.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  link: http://railsgirls.com/brussels
+-
+  date: 31 Dec 1999
+  title: Brug on the News
+  content: >
+    Can't get enough of Ruby? Come code with us.
+  author: Silvana
+  author_image: silvana.jpg
+  press_logo: rubyburgers.jpg
+  link: http://www.meetup.com/fr-FR/ruby_belgium/
```
Comment (borisrorsvort) [inline data/press.yml]: We cannot merge not real content so this will  need to be discussed.

File: source/_navigation.haml [hunk 5]
```diff
@@ -24,4 +24,6 @@
         %li
           %a.page-scroll{href: "/#contact"} Contact
         %li
+          %a.page-scroll{href: "/press"} Press Kit
```
Comment (borisrorsvort) [inline source/_navigation.haml]: I'm thinking that this page might be merged with contact. Not sure yet

File: source/assets/stylesheets/components/_cards.scss [hunk 6]
```diff
@@ -0,0 +1,156 @@
+// Variables
+
+$light:  #febbbb;
+$medium: #fe9090;
+$dark:   #ff5c5c;
+$dark-v1: #c84848;
+$text-color: #999;
+$cta: #00c5a7;
+$card-font-small: 15px;
+
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: Not sure you need all these color as bootstrap already give you a lot.

File: source/assets/stylesheets/components/_cards.scss [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+  background: $light;
+  background: linear-gradient(135deg, $light 0%,$medium 45%,$dark 100%);
+  font-size: 10px;
+  padding: 2em;
+}
+.full-screen {
+  height: 100vw;
+  position: relative;
+}*/
+
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: Please dont commit commented code

File: source/assets/stylesheets/components/_cards.scss [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+
+// General content
+.title {
+  text-align: center;
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: This is too generic to be un-scoped. It has not been decided yet what will be the convention for naming but I'm personnaly using BEM convention for years and its has been proven efficient and widely used.

File: source/assets/stylesheets/components/_cards.scss [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+  font-weight: 100;
+}
+.divider {
+  border-bottom: 1px solid $light;
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: same here. also why not used <hr> ?

File: source/assets/stylesheets/components/_cards.scss [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    .info {
+      transition: top 0.5s ease, opacity 1s ease;
+      visibility: visible;
+      top: 1%;
+      opacity: 1;
+      .divider {
+        /*//opacity: 1;*/
+      }
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: empty css rule

File: source/assets/stylesheets/components/_cards.scss [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+
+    .font-small {
+      font-size: $card-font-small;
+    }
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: usually bootstrap make use of the `small` html tag to do this

File: source/assets/stylesheets/components/_cards.scss [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+    .icon-right {
+      font-size: 2em;
+      right: 45%;
+      transition: font-size .3s, right .3s;
+    }
+  }
+
+
+  .info {
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: too generic naming to be alone in the wild

File: source/assets/stylesheets/components/_cards.scss [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+
+  .lead {
+    color: white; /*$text-color;*/
+    font-weight: 50;
+    font-size: 2em;
+    text-align: center;
+    hyphens: auto;
+  }
+
+  .press-btn {
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: Can't you use nomal bootstrap btn styles for this ?

File: source/assets/stylesheets/components/_cards.scss (hunk 2 above)
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: What do you mean by scaling them ? Usually setting a height on responsive website element is a bad idea

File: source/press/index.haml [hunk 14]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (borisrorsvort) [inline source/press/index.haml]: I'm thinking that this page might be merged with contact. Not sure yet

File: source/assets/stylesheets/components/_cards.scss [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+  background: #f2f2f2;
+  border-radius: 3px;
+  cursor: pointer;
+  height: 17em;
+  padding: 1.5em;
+  position: relative;
+  transition: height 0.4s, box-shadow 0.5s;
+
+  &:hover {
+    box-shadow: 0 0 2em #999;
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: This is very bold visual change, not really in phase with the new theme. You'll see soon

File: source/assets/stylesheets/components/_cards.scss [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+    height: 38em;
```
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: Same remarks about height

File: Gemfile.lock (hunk 3 above)
Comment (dantesolis) [inline Gemfile.lock]: That was a typo probably.

File: source/assets/stylesheets/components/_cards.scss (hunk 2 above)
Comment (dantesolis) [inline source/assets/stylesheets/components/_cards.scss]: The content comes out of my cards when I shrink the browser window.
presspage3

File: Gemfile.lock (hunk 3 above)
Comment (borisrorsvort) [inline Gemfile.lock]: Nope you probably have an old version of bundler

File: data/press.yml (hunk 4 above)
Comment (dantesolis) [inline data/press.yml]: I didn't know what other content to put, thought about using `lorem ipsulum....` but wanted to tested with "fake-real" data, should I had use `lorem ipsulum` instead?

File: source/assets/stylesheets/components/_cards.scss (hunk 2 above)
Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: Why would you keep 3 columns on such a small screen anyway? Boostrap has classes stock them on mobile. Check the portfolio items

File: data/press.yml (hunk 4 above)
Comment (borisrorsvort) [inline data/press.yml]: But then do we need that page?

File: source/assets/stylesheets/components/_cards.scss (hunk 6 above)
Comment (dantesolis) [inline source/assets/stylesheets/components/_cards.scss]: Should I use those in the ones in `_variables.scss` instead?

Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: Yes but those are already overwrite of https://getbootstrap.com/customize/

File: source/assets/stylesheets/components/_cards.scss (hunk 9 above)
Comment (dantesolis) [inline source/assets/stylesheets/components/_cards.scss]: wanted to give the `hr` another color, but you're right it makes it more complicated than what it should be.

File: source/assets/stylesheets/components/_cards.scss (hunk 12 above)
Comment (dantesolis) [inline source/assets/stylesheets/components/_cards.scss]: Was thinking about doing `.img-center {}`, but was afraid of overriding  the `img {}` selector or creating a new unnecessary one. Should I change them to `img-center` and `img-right` right respectively?

Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: img are inline elements so .text-center works other wise you can use .mx-auto

File: source/assets/stylesheets/components/_cards.scss (hunk 16 above)
Comment (dantesolis) [inline source/assets/stylesheets/components/_cards.scss]: How do you go into setting the height of the content, when I don't set it like that. They cards reajust depending on the content. Sorry, still extremely new to css. 
q

Comment (borisrorsvort) [inline source/assets/stylesheets/components/_cards.scss]: Yeah exactly, that's the point, they should readjust

**Comment by dantesolis** [comment]
Hey @borisrorsvort I saw that this pull request was closed, I'm still working on the press page. I still want to contribute. Sorry that I've been a way for a while, just started a new job and have been busy with that.

So what's the best way to proceed? Both with press page -  aside from what was discussed - and contributing in general.

Thanks.

**Comment by borisrorsvort** [comment]
@dantesolis From our last conversation and last comment of toch in https://github.com/brug-be/rubybelgium/issues/27 we think there is no purpose for a press page for now. 

This is why I asked you if you could gather press articles first and then we would see where it make sense to display them. But this is not urgent.

My bad for the lack of clarity. I should have closed the ticket a long time ago. 

For contribution you can have a look at the "help wanted" label in the ticket list https://github.com/brug-be/rubybelgium/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

Make sure you ask for clarification if the ticket description is too broad

**Pull request closed without merging**